script.js

```diff
@@ -2,18 +2,198 @@ const myItem = document.getElementById('menuitems');
+projectPop.appendChild(myImage);
+
+const myProjectDes = document.createElement('p');
+myProjectDes.classList = 'myPop_des';
+myProjectDes.innerText = `${projectDatiels[0].projectText}`;
+projectPop.appendChild(myProjectDes);
+
+const ul1 = document.createElement('ul');
+ul1.classList = 'myCode_lan';
+projectPop.appendChild(ul1);
+
+const listLang1 = document.createElement('li');
+listLang1.classList = 'lang_list';
+listLang1.innerText = `${projectDatiels[0].projectLanguages[0]}`;
+ul1.appendChild(listLang1);
+
+const listLang2 = document.createElement('li');
+listLang2.classList = 'lang_list';
+listLang2.innerText = `${projectDatiels[0].projectLanguages[1]}`;
+ul1.appendChild(listLang2);
+
+const listLang3 = document.createElement('li');
+listLang3.classList = 'lang_list';
+listLang3.innerText = `${projectDatiels[0].projectLanguages[2]}`;
+ul1.appendChild(listLang3);
+
+const myButtons = document.createElement('div');
+myButtons.classList = 'my_btns';
+projectPop.appendChild(myButtons);
+
+const a = document.createElement('a');
+a.classList = 'pop_button';
+a.innerText = 'See Live';
+a.href = `${projectDatiels[0].projectLive}`;
+myButtons.appendChild(a);
+
+const a1 = document.createElement('a');
+a1.classList = 'pop_button';
+a1.innerText = 'See Source';
+a1.href = `${projectDatiels[0].projectLive}`;
+myButtons.appendChild(a1);
+
+const myPopProjects = document.querySelectorAll('.bttn');
+const myPopBox = document.getElementById('myPopFull');
+const popMenuClose = document.getElementById('close_menu1');
+
+function popBox() {
+  if (myPopBox.style.display === 'none') {
+    myPopBox.style.display = 'block';
+    myImage.src = `${projectDatiels[0].projectImage}`;
+    textInner.innerText = `${projectDatiels[0].name}`;
+    window.scroll(0, 1);
   } else {
-    myItem.style.display = 'none';
+    myPopBox.style.display = 'none';
   }
 }
-myMenu.addEventListener('click', mobileMenu);
+
+myPopProjects[0].addEventListener('click', popBox);
+
+function popBox1() {
+  if (myPopBox.style.display === 'none') {
+    myPopBox.style.display = 'block';
+    myImage.src = `${projectDatiels[2].projectImage}`;
+    textInner.innerText = `${projectDatiels[2].name}`;
+    window.scroll(0, 1);
+  } else {
+    myPopBox.style.display = 'none';
+  }
+}
+
+myPopProjects[1].addEventListener('click', popBox1);
+
+function popBox2() {
+  if (myPopBox.style.display === 'none') {
+    myPopBox.style.display = 'block';
+    myImage.src = `${projectDatiels[3].projectImage}`;
+    textInner.innerText = `${projectDatiels[3].name}`;
+    window.scroll(0, 1);
+  } else {
+    myPopBox.style.display = 'none';
+  }
+}
+
+myPopProjects[2].addEventListener('click', popBox2);
+
+function popBox3() {
+  if (myPopBox.style.display === 'none') {
+    myPopBox.style.display = 'block';
+    myImage.src = `${projectDatiels[4].projectImage}`;
+    textInner.innerText = `${projectDatiels[4].name}`;
+    window.scroll(0, 1);
+  } else {
+    myPopBox.style.display = 'none';
+  }
+}
```

Comment:
[OPTIONAL] After studying your code I understand that you are using each of these functions to create a popup. Imagine a scenario where your portfolio becomes bigger and u have about 12 projects to add. Will you write 12 individual functions for each card? 😄 

Hell no, that will be too burdensome, so instead, I recommend the usage of a loop in creating these cards so that you will not have to repeat yourself and the javascript loop is there to help you do things repeatedly.


Consider this:

```
projectDetails.forEach((item) => {
  create each card.

if the items in projectDetails are 12, then 12 cards are created.
})

```